src/picturepark-sdk-v1-angular/projects/picturepark-sdk-v1-angular-ui/src/lib/features-module/layer-panels/models/layer-field.ts

```diff
@@ -23,6 +25,7 @@ export class LayerField {
     this.fieldsetFields = [];
     this.relatedField = field;
     this.name = field.names && field.names['x-default'];
```

Comment:
Can't we just use the LocalizationService here?

Reply:
Ended up using the Translate pipe instead as discussed